site/en/guides/porting/set-up-the-build-environment.md

```diff
@@ -1,122 +1,143 @@
 # Set Up the Build Environment
 
-To promote free and open development, OpenThread uses GNU Autotools in the build
-toolchain. Currently, this toolchain is required for porting OpenThread to a new
-hardware platform.
+To promote free and open development, OpenThread uses [CMake][cmake-homepage] in
+the build toolchain. Currently, this toolchain is required for porting
+OpenThread to a new hardware platform.
 
 Other build toolchains might be supported in the future, but they are not within
 the scope of this porting guide.
 
-> Note: For all path and code examples in this porting
-guide, always replace {platform-name} with the name of
-your new platform example. Most of the examples in the guide use a `efr32` platform
-name.
+> Note: For all path and code examples in this porting guide, always replace
+`{platform-name}` with the name of your new platform example. Most of the
+examples in the guide use a `efr32` platform name.
 
-## Step 1: GNU Autoconf
+[cmake-homepage]: https://cmake.org/
 
-The [Autoconf](https://www.gnu.org/software/autoconf/autoconf.html) script
-contains the basic system configuration options, including specific
-platform-relative macro definitions. These macros can be exposed for
-conditional compilation in other Makefiles during the pre-compiling phase.
+## Step 1: Create a new repository
 
-The OpenThread Autoconf script is located at:
-[`/openthread/configure.ac`](https://github.com/openthread/openthread/blob/main/configure.ac)
+The first step is set up a new home for your hardware platform. In this guide,
+we'll be creating a new repo named `ot-efr32` which will house the
+platform-abstraction layer, the hardware platform's SDK, and a few useful
+scripts.
 
-### Platform example name
+> *TODO* : Do we want to demonstrate making a new git repo through GitHub?
```

Comment:
Do we want to demonstrate making a new git repo through GitHub?

Reply:
I think saying just create a new repo is sufficient. I don't think we need to show the GitHub specifics.